app/components/pages/RewardsCatalogSettings/tests/RewardsCatalogSettings.test.js

```
@@ -37,7 +43,19 @@ describe('<RewardsCatalogSettings />', () => {
   };
 
   const renderWithRouter = (component) => {
-    return render(<BrowserRouter>{component}</BrowserRouter>);
+    let history = createMemoryHistory();
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'history' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)